frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/grid_page.dart

```diff
@@ -221,28 +213,29 @@ class _GridRowsState extends State<_GridRows> {
               },
             );
           },
-          buildWhen: (previous, current) {
-            return current.reason.maybeWhen(
-              reorderRows: () => true,
-              reorderSingleRow: (reorderRow, rowInfo) => true,
-              delete: (item) => true,
-              insert: (item) => true,
-              orElse: () => false,
-            );
-          },
+          buildWhen: (previous, current) => current.reason.maybeWhen(
+            reorderRows: () => true,
+            reorderSingleRow: (reorderRow, rowInfo) => true,
+            delete: (item) => true,
+            insert: (item) => true,
+            orElse: () => false,
+          ),
           builder: (context, state) {
             final rowInfos = context.watch<GridBloc>().state.rowInfos;
 
-            return SliverFillRemaining(
+            return ScrollConfiguration(
+              behavior:
+                  ScrollConfiguration.of(context).copyWith(scrollbars: false),
               child: ReorderableListView.builder(
+                /// This is a workaround related to
+                /// https://github.com/flutter/flutter/issues/25652
+                cacheExtent: 5000,
```

Comment:
Why'd you choose 5000?

Reply:
Honestly I didn't even want to use cacheExtent, I think it's a really bad solution, but I couldn't find any other good ways to eliminate the inconsistent scrollbar behavior without spending too much time.

The number 5000 is half arbitrary, but it's approximately an empty cell * 110 (+ a bit).

I should probably add a todo here, so we can look into fixing it at a later time. The inconsistency occurs because a ListView isn't rendered all at once, so the Scrollbar needs to know the extent of each cell, and believes them to be consistently the same. That means once a cell with a different extent (height) renders, the scrollbar is confused and thus changes position (correctly) and becomes smaller (correctly).

I have some solutions in mind, but let's discuss those at a later time.